src/www/electron.webpack.js

```diff
@@ -54,7 +54,8 @@ module.exports = makeConfig({
     // @sentry/electron depends on electron code, even though it's never activated
     // in the browser. Webpack still tries to build it, but fails with missing APIs.
     // The IgnorePlugin prevents the compilation of the electron dependency.
-    new webpack.IgnorePlugin({resourceRegExp: /^electron$/}),
+    new webpack.IgnorePlugin(
+        {resourceRegExp: /^electron$/, contextRegExp: /@sentry\/electron/}),
```

Comment:
Should this be `/^@sentry...`?

Reply:
From the [sample here](https://webpack.js.org/plugins/ignore-plugin/#example-of-ignoring-moment-locales) it seems the "context" should be ended with `$` and the "resource" should be wrapped in `^...$`.

Reply:
Yeah - from the example - `any directories ending with 'moment' will be ignored`. `@sentry/electron` has subdirectories so we want to make sure to ignore all of them